src/style.css

```
@@ -0,0 +1,78 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-size: normal;
+  font-weight: bold;
+}
+
+body {
+  margin-top: 20px;
+
+}
```

```suggestion
body {
  margin-top: 20px;
}
```
add this margin to form